### ISSUE TITLE
The adjustment of learning rate

### ISSUE THREAD

**Issue by cici-999** [zhanggang001/RefineMask#33]
Thanks for the nice work. I have a question for you. Your default is 8 cards, and you set the learning rate to 0.02. If I conduct the experiment from multiple cards to single card, does the learning rate need to be adjusted? I have tried to adjust the learning rate to 0.0025 and 0.005 for experiments, but the results are not satisfactory. May I ask if you have any suggestions on the adjustment of learning rate when switching from multiple cards to single cards? Looking forward to your reply.

**Comment by zhanggang001** [comment]
The learning rate is set to 0.02 for a total batch size of 16 in RefineMask. If you use a batch size smaller/larger than 16, please follow the linear scaling rule to adjust the learning rate (i.e. 16/8/4 - 0.02/0.01/0.005). However, If you use a batch size smaller than 16, the performance will drop in MMDetection.

**Comment by cici-999** [comment]
> 

Thanks for your reply. I have tried to set samples_per-GPU to 2 on R50-Refinemask-1x with a single card and a learning rate of 0.0025 using coco data set. The result is 36.0, which is worse than the result in your paper. If I use coco dataset and set samples_per-GPU to 1 in R50-Refinemask-1x, single card and learning rate to 0.00125. Is that possible?

**Comment by zhanggang001** [comment]
> > 
> 
> Thanks for your reply. I have tried to set samples_per-GPU to 2 on R50-Refinemask-1x with a single card and a learning rate of 0.0025 using coco data set. The result is 36.0, which is worse than the result in your paper. If I use coco dataset and set samples_per-GPU to 1 in R50-Refinemask-1x, single card and learning rate to 0.00125. Is that possible?

The critical thing is the total training batch size can not be smaller than 16, and you can use 16x1 or 8x2 to reproduce the results presented in the paper.
